webapp/src/pages/createApplicationForm/components/Question.js

```diff
@@ -359,9 +567,61 @@ const Question = forwardRef(({
                   ))}
                 </tbody>
               </table>
+              <span className='error-label'>{t(errorMessage)}</span>
           </div>
           )}
-          
+          {withReferals.includes(inputs.type && inputs.type.value) && (
+            <div className='referals-wrapper'>
+              <input
+                type='number'
+                min={0}
+                placeholder={t('Enter Minum Number of Referrals')}
+                className='referals-input'
+                value={inputs.options.min_num_referral}
+                onChange={handlereferralsChange('min')}
+               />
+              <input
+                type='number'
+                min={0}
+                placeholder={t('Enter Maximum Number of Referrals')}
```

Comment:
One issue with using placeholders here instead of labels is that when the user comes to edit an existing form at some point, the placeholder won't be visible and it's not clear what these values refer to. This applies in a few cases, so although it may not look as clean, it's actually better UX to use always-visible labels in cases like this...